mmdet/datasets/builder.py

```diff
@@ -206,10 +204,15 @@ def build_dataloader(dataset,
     return data_loader
 
 
-def worker_init_fn(worker_id, num_workers, rank, seed):
+def worker_init_fn(worker_id, num_workers, rank):
```

Comment:
seems that the user seed is not used and the interface is changed. Try to avoid BC-breaking of the interface.